server.js

```diff
@@ -0,0 +1,78 @@
+const { client, syncAndSeed } = require ('./db')
+const express = require ('express')
+const path = require ('path')
```

Comment:
i know prof doesnt go through this but general in software engineering, library imports go on top, local imports go below it, separated by a space like this- 
```suggestion
const express = require ('express')
const path = require ('path')

const { client, syncAndSeed } = require ('./db')
```